README.md

```diff
@@ -1,5 +1,13 @@
 # Graasp Apps Query Client
 
+[![npm (scoped)](https://img.shields.io/npm/v/@graasp/apps-query-client?logo=npm)](https://www.npmjs.com/package/@graasp/apps-query-client)
+[![GitHub package.json version](https://img.shields.io/github/package-json/v/graasp/graasp-apps-query-client?color=deepskyblue&logo=github)](https://github.com/graasp/graasp-apps-query-client/releases/latest)
+![npm](https://img.shields.io/npm/dm/@graasp/apps-query-client?color=green)
+
+![GitHub package.json dependency version (dev dep on branch)](https://img.shields.io/github/package-json/dependency-version/graasp/graasp-apps-query-client/dev/typescript)
+![npm peer dependency version (scoped)](https://img.shields.io/npm/dependency-version/@graasp/apps-query-client/peer/react?logo=react)
+![GitHub package.json dependency version (prod)](https://img.shields.io/github/package-json/dependency-version/graasp/graasp-apps-query-client/react-query?logo=react-query)
```

Comment:
The alternative texts does not mention which package the version is referring to.